testing/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@stencil/core",
+  "version": "0.0.1",
+  "license": "MIT",
+  "main": "../dist/testing/index.js",
+  "types": "../dist/testing/index.d.ts",
```

Comment:
Rather than having `testing.*` at the root level (original code) or `testing/index.*` that needed to potentially be maintained, this just points to the `dist/testing/index.*` files which are just built from the source as part of the build process.